src/Uno.UWP/Storage/Pickers/FileOpenPicker.Android.cs

```diff
@@ -99,10 +99,36 @@ private string[] GetMimeTypes()
 				return new[] { "*/*" };
 			}
 
-			return FileTypeFilter
-				.Select(extension => MimeTypeService.GetFromExtension(extension))
-				.Distinct()
-				.ToArray();
+			List<string> mimeTypes = new List<string>();
+			foreach(string oneExtensionForLoop in FileTypeFilter)

```

Comment:
Can you make sure the indentation is correct?

Reply:
Probably resolved as side-affect of commit update.